GPU/Vulkan/TextureScalerVulkan.cpp

```diff
@@ -21,38 +21,45 @@
 #endif
 
 #include <algorithm>
-#include "gfx/gl_common.h"
 
-#include "GPU/Common/TextureScalerCommon.h"
-#include "GPU/Vulkan/TextureScalerVulkan.h"
+#include "Common/Vulkan/VulkanContext.h"
 #include "Common/ColorConv.h"
 #include "Common/Log.h"
 #include "Common/ThreadPools.h"
+#include "GPU/Common/TextureScalerCommon.h"
+#include "GPU/Vulkan/TextureScalerVulkan.h"
+
+// TODO: Share in TextureCacheVulkan.h?
```

Comment:
Of course, I meant expose it in the TextureCacheVulkan header.  Maybe utils makes sense... it's just for the cases.

-[Unknown]